ci/create_database.py

```diff
@@ -126,6 +149,8 @@ def generate_token(size=12):
     current_version = row['version']
 
     if current_version + 1 == to_version:
+        shutdown()
```

Comment:
await

Reply:
Argh, thanks.  I would give a lobe of my liver for a linter that checks asyncio errors.

Reply:
seconded.